### PR TITLE
update moonriver vuex-options

### PR DESCRIPTION
**Pull request by zhengow**
[update moonriver vuex-options](https://github.com/kodadot/nft-gallery/issues/3177#issuecomment-1159675810)

### CODE REVIEW

**Comment by vikiival** [comment]
> No description provided.

For the future pls

File: vuex-options/src/defaults/endpoints.ts [hunk 1]
```diff
@@ -64,6 +64,11 @@ export const ENDPOINTS: Option[] = [
     text: 'Dusty (Stake Technologies)',
     value: 'wss://rpc.dusty.plasmnet.io'
   },
+  {
+    info: 'moonriver',
+    text: 'MoonRiver(Moonbeam)',
```
Comment (vikiival) [inline vuex-options/src/defaults/endpoints.ts:69]:
```suggestion
    text: 'Moonriver (Moonbeam)',
```

File: vuex-options/src/defaults/indexers.ts [hunk 2]
```diff
@@ -34,6 +34,11 @@ export const INDEXERS: Option[] = [
     text: 'Westmint (unique-west)',
     value: 'https://api.subquery.network/sq/vikiival/unique-west'
   },
+  {
+    info: 'moonriver',
+    text: 'MoonRiver(Moonbeam)',
```
Comment (vikiival) [inline vuex-options/src/defaults/indexers.ts:39]:
```suggestion
    text: 'Moonriver (click)',
```

**Review by vikiival — CHANGES_REQUESTED** [review]

**Comment by vikiival** [comment]
Please also update package version

**Comment by zhengow** [comment]
> Please also update package version

done:)

File: vuex-options/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kodadot1/vuex-options",
-  "version": "0.0.1-rc.7",
+  "version": "0.0.1-rc.8",
```
Comment (vikiival) [inline vuex-options/package.json:3]:
```suggestion
  "version": "0.0.1-rc.9",
```

**Review by vikiival — APPROVED** [review]

File: vuex-options/package.json (hunk 3 above)
Comment (zhengow) [reply vuex-options/package.json]: Actually, I update 0.0.1-rc.7 to 0.0.1-rc.8 for my commit.
So 0.0.1-rc.8 to 0.0.1-rc.9 update nothing 😂

**Pull request merged**